components/brave_wallet/browser/brave_wallet_service.cc

```diff
@@ -1440,7 +1466,7 @@ void BraveWalletService::NotifyAddSuggestTokenRequestsProcessed(
     bool approved,
     const std::vector<std::string>& contract_addresses) {
   const std::string chain_id =
-      GetCurrentChainId(profile_prefs_, mojom::CoinType::ETH);
+      GetCurrentChainId(profile_prefs_, mojom::CoinType::ETH, absl::nullopt);
```

Comment:
chain_id is actually unused, please remove this instead, chain_id info is already in the token that's going to be added.

Reply:
fixed in 833fccfc137c8bd32c0df6c87e44dd9861dbc9e4
Unused chain_id is probably caused by rebase